internal/services/postgres/postgresql_server_resource.go

```diff
@@ -360,6 +358,22 @@ func resourcePostgreSQLServer() *pluginsdk.Resource {
 			}),
 		),
 	}
+
+	if !features.FivePointOhBeta() {
+		resource.Schema["ssl_minimal_tls_version_enforced"] = &pluginsdk.Schema{
+			Type:         pluginsdk.TypeString,
+			Optional:     true,
+			Default:      string(servers.MinimalTlsVersionEnumTLSOneTwo),
+			ValidateFunc: validation.StringInSlice(servers.PossibleValuesForMinimalTlsVersionEnum(), false),
+		}
+		resource.Schema["ssl_enforcement_enabled"] = &pluginsdk.Schema{
+			Deprecated: "The `ssl_enforcement_enabled` is deprecated as Azure services will require TLS1.2+ to connect.",
+			Type:       pluginsdk.TypeBool,
+			Required:   true,
```

Comment:
This probably wants to become `Optional` so users can begin removing it from their configs
```suggestion
			Optional:   true,
```